build/cqs.tsv

```diff
@@ -144,6 +144,8 @@ org.apache.curator:curator-recipes	2.4.0	compile
 org.apache.curator:curator-recipes	2.7.1	compile
 org.apache.hbase:hbase-client	1.3.1	compile
 org.apache.hbase:hbase-common	1.3.1	compile
+org.apache.hbase:hbase-hadoop2-compat	1.3.1	compile
+org.apache.hbase:hbase-hadoop-compat	1.3.1	compile
```

Comment:
we'll need to double check these are covered with our current cq

Reply:
I don't think I have access to that system yet.

Reply:
I'm not sure we actually need the change - it was just for the gs-plugin right? I think this may have been left over from hbase 1.4 testing. If we need it, I can check the CQ

Reply:
Hmm, this is probably from the hbase 1.4 / aws testing. Probably fine without.